lib/pure/os.nim

```diff
@@ -944,17 +944,32 @@ proc rawRemoveDir(dir: string) =
   else:
     if rmdir(dir) != 0'i32 and errno != ENOENT: raiseOSError(osLastError())
 
-proc removeDir*(dir: string) {.rtl, extern: "nos$1", tags: [
+proc removeDir*(dir: string, recursive: bool = true) {.raises: [IOError, OSError], rtl, extern: "nos$1", tags: [
```

Comment:
You don't need to specify the type if you've specified a default value.

Reply:
I normally do it just to be 100 percent clear in every situation, but this does seem excessive, I'll change this as well, thanks!